vllm/entrypoints/openai/api_server.py

```diff
@@ -126,7 +245,6 @@ async def create_completion(request: CompletionRequest, raw_request: Request):
     else:
         return JSONResponse(content=generator.model_dump())
 
-
```

Comment:
Please avoid these unnecessary changes in the file in the future commits

Reply:
> Please avoid these unnecessary changes in the file in the future commits

Ok I will pay attention to it.